packages/story-editor/src/app/story/effects/useHistoryEntry.js

```diff
@@ -23,28 +23,83 @@ import { useEffect, useRef } from '@web-stories-wp/react';
  * Internal dependencies
  */
 import { useHistory } from '../../history';
+import deleteNestedKeys from '../utils/deleteNestedKeys';
+
+// @todo REMOVE ROTATION ANGLE! For testing only!
+const ELEMENT_PROPS_TO_IGNORE = ['resource.baseColor', 'rotationAngle'];
```

Comment:
I would also add `resource.isMuted`, `resource.poster` and `resource.posterId`, as these are also generated in the background. 

We may also want to think about `resource.src`, which is updated from a blob to a url which an update is completed. 

It is also work noting that isTrimming, isMutting and isTrancoding properties are being removed as part of https://github.com/google/web-stories-wp/pull/9785

Reply:
Thanks for these, any others from media elements you can think of that are mutating history?